go.mod

```diff
@@ -7,5 +7,5 @@ require (
 	github.com/casbin/gorm-adapter/v2 v2.0.0
 	github.com/golang/protobuf v1.3.2
 	github.com/stretchr/testify v1.3.0
-	google.golang.org/grpc v1.23.0
+	google.golang.org/grpc v1.27.0
```

Comment:
Why `go.sum` is not updated?

Reply:
Missed the file. I have pushed the updated file.